recipe/build.sh

```diff
@@ -1,7 +1,18 @@
 #!/bin/bash
 
+export CMAKE_CONFIG="Release"
 
-mkdir build
-cd build
-cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} -GNinja ..
-ninja install
+mkdir "build_${CMAKE_CONFIG}"
+pushd "build_${CMAKE_CONFIG}"
+
+cmake ${CMAKE_ARGS} \
+      -G Ninja \
+      -D CMAKE_BUILD_TYPE:STRING="${CMAKE_CONFIG}" \
+      -D CMAKE_PREFIX_PATH:PATH="${PREFIX}" \
+      -D CMAKE_INSTALL_PREFIX:PATH="${PREFIX}" \
+      "${SRC_DIR}" \
+;
+cmake --build . --parallel ${CPU_COUNT}
```

Comment:
Just curious, is there a reason/benefit for using `--parallel` instead of `-GNinja`?

Reply:
We are still using `-G Ninja` in the previous invocation of `cmake`. This command generalizes over commands like `make` or `ninja` (without needing to change should the CMake generator change)

Reply:
Ah right, my bad, I overlooked that. Thanks for pointing that out!

Reply:
All good. It is syntax not everyone is familiar with